indexer/services/roundtable/__tests__/tasks/market-updater.test.ts

```diff
@@ -65,7 +130,7 @@ describe('market-updater', () => {
   });
 
   beforeEach(async () => {
-    await testMocks.seedData();
+    await seedData();
```

Comment:
The sparklines-controller test requires the return of the entire universe of perpetual markets. Its very annoying to use the testMocks seed data which is shared between all tests since adding something simple like a new perpetual means you have to change this test
For tests such as these, its better to do the seed the data within the test itself